launcher/ui/pages/global/APIPage.ui

```diff
@@ -195,6 +195,51 @@
          </layout>
         </widget>
        </item>
+       <item>
```

Comment:
With your modifications I get the following warnings when compiling:
```
[...]/APIPage.ui: Warning: The name 'gridLayout' (QGridLayout) is already in use, defaulting to 'gridLayout1'.
[...]/APIPage.ui: Warning: The name 'label_8' (QLabel) is already in use, defaulting to 'label_81'.
[...]/APIPage.ui: Warning: The name 'label_7' (QLabel) is already in use, defaulting to 'label_71'.
```
might be worth it to change those to keep the build logs cleaner :-)

Reply:
are the names needed anyway? removing them is possibly easier

Reply:
I think objects need names so that they can show up in the Qt Designer thingy at the very least, but I never tested it :thinking: